system/Database/SQLite3/Table.php

```diff
@@ -273,6 +327,14 @@ protected function createTable()
             }
         }
 
```

Comment:
This part fixes the Table class and actually allows the creation or recreation of foreign keys